torchvision/prototype/transforms/_augment.py

```diff
@@ -169,17 +170,18 @@ def _get_params(self, sample: Any) -> Dict[str, Any]:
         return dict(box=box, lam_adjusted=lam_adjusted)
 
     def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
-        if isinstance(inpt, features.Image) or features.is_simple_tensor(inpt):
+        if isinstance(inpt, (features.Image, features.Video)) or features.is_simple_tensor(inpt):
             box = params["box"]
-            if inpt.ndim < 4:
-                raise ValueError("Need a batch of images")
+            expected_dim = 5 if isinstance(inpt, features.Video) else 4
+            if inpt.ndim != expected_dim:
+                raise ValueError("The transform expects a batched input")
```

Comment:
Maybe we can have a small helper method to keep both `CutMix` and `MixUp` in sync here? Like `_check_image_or_video`.

Reply:
IDK. Is it really worth saving 3 lines and introducing more private methods in the base class?